store/sqlstore/supplier.go

```diff
@@ -159,7 +159,7 @@ func NewSqlSupplier(settings model.SqlSettings, metrics einterfaces.MetricsInter
 
 	err := supplier.GetMaster().CreateTablesIfNotExists()
 	if err != nil {
-		mlog.Critical(fmt.Sprintf("Error creating database tables: %v", err))
+		mlog.Critical("Error creating database tables:", mlog.Err(err))
```

Comment:
I would remove the exta `:` and would replace it with `.` for all the cases in the PR.